extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -759,10 +767,12 @@ angular.module('oppia').component('svgFilenameEditor', {
             ctrl.polyOptions.bboxPoints.push(new polyPoint(x, y));
             var points = [x, y, x, y];
             var size = ctrl.fabricjsOptions.size;
+            var stroke = ctrl.fabricjsOptions.stroke;
+            stroke = stroke.slice(0, -2) + '1)';
```

Comment:
Why are we doing this?

Reply:
When drawing a polygon, the initial lines uses stroke color. So if the alpha is zero you won't see the initial diagram, So Iam forcing it to 1 until the polygon is complete. So once the polygon is complete it retains the actual value.

Reply:
I will add a comment in the code.

Reply:
Sounds good. Thanks!

In general, it's always good to add a comment if you get a comment asking why something is the way it is. See tip [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change):
>Tip: If a reviewer asks questions about the "why" behind something, consider proactively adding a clear comment above the relevant line in your code, since the fact that the reviewer had to ask suggests that at least one developer doesn't understand what is going on from the code alone. Otherwise, you'll probably get a follow-up review comment asking you to leave a code comment anyway :)

Reply:
Done.